workflows/sars-cov-2-variant-calling/sars-cov-2-ont-artic-variant-calling/ont-artic-variation.ga

```diff
@@ -888,135 +906,119 @@
         },
         "17": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_find_and_replace/1.1.3",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/lofreq_filter/lofreq_filter/2.1.5+galaxy0",
             "errors": null,
             "id": 17,
             "input_connections": {
-                "infile": {
+                "invcf": {
                     "id": 15,
-                    "output_name": "outvcf"
+                    "output_name": "snpeff_output"
                 }
             },
-            "inputs": [],
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool Lofreq filter",
+                    "name": "invcf"
+                }
+            ],
```

Comment:
And the same problem here ... unless this is intentional ?

Reply:
Hmm, this one might be wrong. Need to check.

Reply:
Yeah, got introduced in this PR and seems wrong. The WF is running without issues though.
@mvdbeek Any specific problem to watch out for?